android/assets/jsons/Civ V - Vanilla/Beliefs.json

```diff
@@ -241,6 +241,56 @@
         "name": "World Church",
         "type": "Founder",
         "uniques": ["[+1 Culture] for every [5] global followers [in foreign cities]"]
+    },
+
+    ////////////////////////////////////// Enhancer beliefs ///////////////////////////////////////
+    
+    {
+        "name": "Defender of the Faith",
+        "type": "Enhancer",
+        "uniques": ["[+20]% Strength for [Non-City] units in [Friendly Land]"],
+        // ToDo: Should only be friendly territory of cities that follow this religion
+    },
+    {
+        "name": "Holy Order",
+        "type": "Enhancer",
+        "uniques": ["[Faith] cost of purchasing [Missionary] units [-30]% ", "[Faith] cost of purchasing [Inquisitor] units [-30]% "]
+    },
+    {
+        "name": "Itinerant Preachers",
+        "type": "Enhancer",
+        "uniques": ["Religion naturally spreads to cities [+3] tiles away"]
+    },
+    {
+        "name": "Just War",
+        "type": "Enhancer",
+        "uniques": ["[+20]% Strength for [Non-City] units in [Enemy Land]"],
+        // ToDo: Should only be enemy territory of cities that follow this religion
+    },
+    {
+        "name": "Messiah",
+        "type": "Enhancer",
+        "uniques": ["[+25]% Spread Religion Strength for [Great Prophet] units", "[-25]% Faith cost of generating Great Prophet equivalents"]
+    },
+    {
+        "name": "Missionary Zeal",
+        "type": "Enhancer",
+        "uniques": ["[+25]% Spread Religion Strength for [Missionary] units"]
+    },
+    {
+        "name": "Religious Texts",
+        "type": "Enhancer",
+        "uniques": ["[+34]% Natural religion spread to [in all cities]", "[+34]% Natural religion spread to [in all cities] after discovering [Printing Press]"]
```

Comment:
"Spread to in all cities" throws me parse errors in English

Reply:
I completely agree, but this is a direct result of the way `CityInfo.matchesFilter()` was implemented. There are two options here:
- Change these uniques to not use city-filters and do more hardcoding -- less modularity
- Change the cityFilters themselves, removing the "in" prefix for cases like these -- more work for modders.

Which do you prefer?

Reply:
Religion spread [in all cities]

Reply:
Somehow I was trying to write two things at once, looking at this with a fresh view makes this look really weird, I'll change this :)